sygnal/pipeline.yml

```diff
@@ -0,0 +1,27 @@
+steps:
+  - command:
+      - "python -m pip install black"
+      - "black --check ."
+    label: "Check Code Formatting"
+    plugins:
+      - docker#v3.0.1:
+          image: "python:3.7"
+
+  - command:
+      - "python -m pip install flake8"
+      - "flake8 ."
+    label: "Check Code Style"
+    plugins:
+      - docker#v3.0.1:
+          image: "python:3.7"
```

Comment:
in an ideal world, these would probably be combined into one pipeline, with a `lint.sh` in the sytest repo (since the linting steps should be kept in step with the code rather than being defined separately in the pipeline repo).

Still, that can happen another time.